# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blitz/bzconfig.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- blitz-0.6/compiler/bzconfig.vanilla	2005-09-22 09:22:31.000000000 +0200
+++ blitz-0.6/compiler/bzconfig	2005-09-22 09:22:58.000000000 +0200
@@ -37,11 +37,6 @@
 # to edit the next line
 objextension='o'
 
-if test ! -t 0; then
-    echo "Use 'sh bzconfig', not 'sh < bzconfig'"
-    exit 1
-fi
-
 # clean="rm -f a.out core bztemp bztest bzjunk.cpp bzjunk.o bzjunk.ii bzjunk.int.c bzjunk.s"
 clean="rm -f core"
 trap '$clean; exit 1' 1 2 3 15
